lib/ViewModels/SharePopupViewModel.js

```diff
@@ -12,9 +12,20 @@ var inherit = require('../Core/inherit');
 var ModelError = require('../Models/ModelError');
 var PopupViewModel = require('./PopupViewModel');
 
+/**
+ * A dialog that displays a url to the user reflecting the map's current state, and also displays HTML for embedding
+ * it as an iframe into another site.
+ *
+ * @param options {Object}
+ * @param options.terria {Terria} the terria instance to get data from
+ * @param options.userPropWhiteList {Object} A white list of userProperty ids that will be reflected in the share link.
+ *          Defaults to SharePopupViewModel#defaultUserPropWhiteList. Override this if you have custom userProperties
+ *          that you want to be shared along with the rest of the map.
+ */
 var SharePopupViewModel = function(options) {
     PopupViewModel.call(this, options);
     this.terria = options.terria;
+    this.userPropWhiteList = options.userPropWhiteList || SharePopupViewModel.defaultUserPropWhiteList;
```

Comment:
We usually use `defaultValue(options.usePropWhiteList, SharePopupViewModel.defaultUserPropWhiteList)` (unless the second parameter is expensive).